src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -1890,6 +1898,24 @@ private static String handleKeyword(final String field) {
         return field;
     }
 
+    public static DiscoveryNode getLocalNode() {
+        return localNode;
+    }
+
+    public static void setLocalNode(DiscoveryNode node) {
+        localNode = node;
+    }
```

Comment:
this should be removed. Check this PR for more details: https://github.com/opensearch-project/security/pull/3066/files